client/components/credit-card/stored-card.jsx

```diff
@@ -0,0 +1,49 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+export const cardType = PropTypes.shape( {
+	card: PropTypes.string,
+	card_type: PropTypes.string,
+	name: PropTypes.string,
+	expiry: PropTypes.string,
+} );
+
+class StoredCard extends React.Component {
+	static propTypes = {
+		card: cardType,
+	};
+
+	render() {
+		const card = this.props.card;
+		const expirationDate = this.props.moment( card.expiry ).format( 'MM/YY' );
```

Comment:
I know this was existing, but out of curiosity is this the best way to approach date localization cc @Automattic/i18n ? I know other countries prefer different groupings. Though this example if probably fine for most.

Reply:
Hi! 

There's an [ISO standard](https://en.wikipedia.org/wiki/ISO/IEC_7813) of `MM(\/|\-)YY` for credit cards and the like. So I'd say we shouldn't need to make it available for translation, and can leave it as it is. Also, it matches the format the user must enter at purchase anyway. 

See: https://translate.wordpress.com/projects/wpcom/-all-translated/240095/

I can find any _country_-specific formats, though I'd hazard a guess a say that any varying formats (where they exist) would be specific to a card/institution.